lib/ansible/modules/cloud/amazon/ec2_vpc_net.py

```diff
@@ -19,7 +19,7 @@
     - Create, modify, and terminate AWS virtual private clouds.
 version_added: "2.0"
 author:
-  - Jonathan Davila (@defionscode)
+  - Jonathan Davila
```

Comment:
@defionscode You may want to add changes to [BOTMETA.yml](https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml) instead of individual modules.

For example - 

```yml
$modules/cloud/amazon/ec2_vpc_net.py:
   maintainers: $team_ansible
   ignored: defionscode
```

Reply:
I did a while back and i still get roped in

Reply:
Oh, I didn't see the `ignored` prop is there a more global setting?

Reply:
I believe you should be able to update
```
  $modules/cloud/amazon/:
    ignored: erydo seiffert simplesteph nadirollo tedder joshsouza
    maintainers: $team_aws
```

And add defionscode to the amazon/ ignored list